borg/archiver.py

```diff
@@ -728,6 +738,10 @@ def do_info(self, args, repository, manifest, key, archive, cache):
         stats = archive.calc_stats(cache)
         print('Name:', archive.name)
         print('Fingerprint: %s' % hexlify(archive.id).decode('ascii'))
+
+        if archive.metadata.get(b'comment'):
+            print('Comment:', archive.metadata[b'comment'])
```

Comment:
i'm still thinking whether it is easier if we always print the Comment: entry, even if empty or not present (and default to "" if not present). That might be easier if someone parses this and would not leave questions open whether this borg version can show comments.